spec/factories/data_request_areas.rb

```diff
@@ -0,0 +1,22 @@
+# == Schema Information
+#
+# Table name: data_request_areas
+#
+#  id                     :bigint           not null, primary key
+#  case_id                :bigint           not null
+#  user_id                :bigint           not null
+#  contact_id             :bigint
+#  data_request_area_type :enum             not null
+#  location               :string
+#  created_at             :datetime         not null
+#  updated_at             :datetime         not null
+#
+FactoryBot.define do
+  factory :data_request_area do
+    association :offender_sar_case
+    association :user
+
+    location { Faker::Company.name }
```

Comment:
Maybe better to set the contact instead of the location as that is expected for new data request areas.